src/main/java/org/spongepowered/api/block/BlockSnapshotBuilder.java

```diff
@@ -63,7 +63,7 @@
     BlockSnapshotBuilder position(Vector3i position);
 
     /**
-     * Copies over block data from a {@link Location<World>}.
+     * Copies over block data from a {@link Location}<{@link World}>.
```

Comment:
Wait, this is how you link for javadoc generics? I need to do this in quite a few places.....